workspace/utilities/lib/popup.xsl

```diff
@@ -8,7 +8,7 @@
 	<xsl:param name="attr-content" />
 	<xsl:param name="attr-close-btn" />
 	<xsl:param name="poped" select="false()" />
-	<xsl:param name="close-btn-content" select="'X'" />
+	<xsl:param name="display-close-button" select="true()" />
 	<xsl:param name="close-btn-url" select="concat(/data/params/current-path, '/')" />
 	<xsl:param name="close-btn-is-optional" select="false()" />
```

Comment:
Seems redundant with display-close-button. I know its not the _exact_ same thing but having this option is more confusing than helpful. What do you think? @nitriques @PascalPiche @beaubbe

Reply:
I share the same opinion as @dominic-blain I'm voting for removing it.

Reply:
@fhamon :meat_on_bone:

Reply:
We keep it. Because reasons.